src/roles/common/tasks/check-vsd-license-validity.yml

```diff
@@ -9,4 +9,9 @@
   delegate_to: localhost
   no_log: "{{ lookup('env', 'METROAE_NO_LOG') or 'true' }}"
 
+- name: Fail when VSD License is expired.
```

Comment:
This isn't going to work the way the library is written.  I have a completely different proposal.